### DISCUSSION TITLE
Any word on when 2024 libraries might be available from REV and CTRE?

### DISCUSSION THREAD

**Discussion by mjhealy** [wpilibsuite/2024Beta#2] [Q&A] [answered]
My team (2656; "Quasics") is excited to start testing the Beta-1 build of the WPILib tools.  However, we're somewhat limited in our ability to put together working (as in, _deployable_) software because our robots are predominantly built around Spark MAX motors and Pigeon2 IMUs.

Has there been any discussion with REV and CTRE (or any other common FRC vendors) about when they expect to have their initial beta libraries available?  Are there contact channels that we can be monitoring for news in this area?

Thanks!

**Comment by PeterJohnson** [comment]
I would expect NI and vendor releases to happen early this coming week. This repo is the place to watch for announcements.

**Comment by jfabellera** [comment, marked as answer]
REVLib v2024.0.0 has been released and can be found here: https://github.com/REVrobotics/REV-Software-Binaries/releases/tag/revlib-2024.0.0

**Comment by TylerSeiford** [comment]
For those who didn't catch the update to the read-me, the navX lib is also updated to 2024: https://dev.studica.com/releases/2024/NavX.json
